src/main/java/bio/terra/controller/RepositoryApiController.java

```diff
@@ -96,13 +113,25 @@ public RepositoryApiController(ObjectMapper objectMapper, HttpServletRequest req
         return new ResponseEntity<>(studySummary, HttpStatus.CREATED);
     }
 
+    public ResponseEntity<List<JobModel>> enumerateJobs(Integer offset, Integer limit) {
+        return jobService.enumerateJobs(offset, limit);
+    }
+
+    public ResponseEntity<JobModel> retrieveJob(String jobId) {
+        return jobService.retrieveJob(jobId);
+    }
+
+    public ResponseEntity<Object> retrieveJobResult(String jobId) {
+        return jobService.retrieveJobResult(jobId);
+    }
+
     public <T> T getResponse(String flightId, Class<T> resultClass) {
```

Comment:
Is this method used? I think a copy has moved into JobService.

Reply:
It's used by the createStudy code -- though likely it gets refactored out in Jeremy's latest pr